### PR TITLE
Update email on 500 page error

### PR DESCRIPTION
**Pull request by luismorenolopera**
## Description

This PR changes the message and the link that are displayed when an error 500 is generated.

### CODE REVIEW

File: lms/templates/static_templates/server-error.html [hunk 1]
```diff
@@ -24,10 +24,8 @@ <h1>
                 % if page_content:
                     ${page_content}
                 % else:
-                    ${Text(_('Please wait a few seconds and then reload the page. If the problem persists, please email us at {email}.')).format(
-                    email=HTML('<a href="mailto:{email}">{email}</a>').format(
-                    email=Text(static.get_tech_support_email_address())
-                    )
+                    ${Text(_('Please wait a few seconds and then reload the page. If the problem persists, please visit {support}.')).format(
+                    support=HTML('<a href="http://pearsonadvance.echelp.org/">Support</a>')
```
Comment (diegomillan) [inline lms/templates/static_templates/server-error.html:28]: wouldn't be better to return this link from "static.get_tech_support_email_address()"

**Review by diegomillan — CHANGES_REQUESTED** [review]

**Comment by Squirrel18** [comment]
Why you don't include this change in the theme? @luismorenolopera

File: lms/templates/static_templates/server-error.html (hunk 1 above)
Comment (luismorenolopera) [reply lms/templates/static_templates/server-error.html:28]: Yes, it is better requires placing the link in TECH_SUPPORT_EMAIL

**Comment by luismorenolopera** [comment]
This change will be added to each of the themes

**Pull request closed without merging**